conf/application.conf

```diff
@@ -0,0 +1,16 @@
+# https://www.playframework.com/documentation/latest/Configuration
+play.http.port=9000
+
+auth.url="http://localhost:8080"
+auth.callback="http://localhost:9000/callback"
+auth.secret="set_me"
```

Comment:
There's no fallback. If you specify config.resource, it will use that file, if you don't, it will use this one. This won't work if we accidentally deploy it to prod without the config.resource param.